flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -100,6 +100,18 @@ under the License.
 			</exclusions>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-sql-parser-hive</artifactId>
+			<version>${project.version}</version>
+			<exclusions>
+				<exclusion>
```

Comment:
Why we append a new dependency instead of override the `flink-sql-parser` ?

Reply:
`flink-sql-parser` is a provided dep of `flink-sql-parser-hive`, which means `flink-sql-parser` is not transitive. I suppose this can make the dependencies clearer if we add a 3rd parser in the future. What do you think?

Reply:
I'm fine with that ~